webpack.config.js

```diff
@@ -125,76 +129,32 @@ webpackConfig.module.noParse = [
 ]
 
 // ------------------------------------
-// Pre-Loaders
+// Rules
 // ------------------------------------
-webpackConfig.module.preLoaders = []
-
-// ------------------------------------
-// Loaders
-// ------------------------------------
-webpackConfig.module.loaders = [{
+webpackConfig.module.rules = [{
   //
   // Babel
   //
   test: /\.js$/,
   exclude: /node_modules/,
-  loader: 'babel',
-  query: {
-    cacheDirectory: true,
+  use: {
+    loader: 'babel-loader',
+    options: {
+      cacheDirectory: true,
+    },
   },
 }, {
   //
-  // JSON
+  // SASS
   //
-  test: /\.(json|.*rc)$/,
-  loader: 'json',
+  test: /\.s?css$/,
+  use: ['style-loader', 'css-loader', 'sass-loader'],
+}, {
+  //
+  // Files
+  //
+  test: /\.(eot|ttf|woff|woff2|svg|png)$/,
+  loader: 'file-loader',
 }]
 
-// ----------------------------------------
-// Local Modules
-// ----------------------------------------
-// For faster builds in dev, rely on prebuilt libraries
-// Local modules can still be enabled (ie for offline development)
-// in TEST we need local modules because karma uses a different index.html (no CDNs)
-if (__TEST__ || argv.localModules) {
-  webpackConfig.module.loaders = [
-    ...webpackConfig.module.loaders,
-    {
-      //
-      // SASS
-      //
-      test: /\.s?css$/,
-      loaders: ['style', 'css', 'sass'],
-    }, {
-      //
-      // Files
-      //
-      test: /\.(eot|ttf|woff|woff2|svg|png)$/,
-      loader: 'file',
-    },
-  ]
-} else {
```

Comment:
We still need the else branch here as it is applicable when not in tests.  I'll review it and push an update.